### PR TITLE
1443 (1/3) remove widget from CI/CD + trigger deploy for FHIRConverter

### PR DESCRIPTION
**Pull request by leite08**
Ticket: https://github.com/metriport/metriport-internal/issues/1443

### Dependencies

- Upstream: https://github.com/metriport/metriport/pull/1519
- Downstream: https://github.com/metriport/metriport/pull/1523

### Description

- remove widget from CI/CD
- trigger deploy for FHIRConverter
- miscellaneous updates 

### Testing

- Staging
  - [ ] deploys work
  - downstream has additional test
- Sandbox
  - [ ] deploys work
- Production
  - [ ] deploys work

### Release Plan

- nothing special other than follow dependencies

### CODE REVIEW

File: .github/workflows/_build.yml [hunk 1]
```diff
@@ -74,30 +73,14 @@ jobs:
...
-        if: ${{ inputs.treat-as-package == false }}
-        run: |
-          pwd
-          npm run build:cloud -w packages/shared
-          npm run build:cloud -w packages/commonwell-sdk
-          npm run build:cloud -w packages/api-sdk
-          npm run build:cloud -w packages/ihe-gateway-sdk
-          npm run build:cloud -w packages/carequality-sdk
-          npm run build:cloud -w packages/core
-        working-directory: "./"
```
Comment (Goncharo) [inline .github/workflows/_build.yml:88]: why is this no longer needed?

**Review by Goncharo — APPROVED** [review]
just one Q, otherwise LGTM :shipit:

**Review by RamilGaripov — APPROVED** [review]
🚢 🚢 🚢

File: .github/workflows/_build.yml (hunk 1 above)
Comment (leite08) [reply .github/workflows/_build.yml:88]: We already [build from root before](https://github.com/metriport/metriport/pull/1522/files/b22354446b5b3a66bd8879b1c66b6c9d525e87c5#diff-ebfafa272202ca4d7cf36473df663e9f7a7cf844c549e60b2a6eae0a8be411edR63-R67), I think this is just duplicated and unnecessary work. Will validate when I merge it.

**Pull request merged**